synapse/config/_base.pyi

```diff
@@ -20,7 +18,7 @@ from typing import (
 
 import jinja2
 
-from synapse.config import (
+from synapse.config import (  # noqa: F401
```

Comment:
Do you need to ignore on both the import and usage line?

Reply:
No, we just need it on the import line? Each `noqa` comment is suppressing different errors?

Reply:
TIL that 401 != 881. (I swear those all said 401...)